weaviate/connect/v4.py

```diff
@@ -126,6 +127,10 @@ def __init__(
             _validate_input(_ValidateArgument([dict], "additional_headers", additional_headers))
             self.__additional_headers = additional_headers
             for key, value in additional_headers.items():
+                if value is None:
+                    raise WeaviateInvalidInputError(
+                        f"Value for key '{key}' in additional_headers cannot be None."
```

Comment:
`additional_headers` in the error, but it is `headers` in the connect_with(..) command